src/Analyzers/Core/CodeFixes/AliasAmbiguousType/AbstractAliasAmbiguousTypeCodeFixProvider.cs

```diff
@@ -61,8 +59,10 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
                         }));
                 }
 
-                var groupingTitle = string.Format(FeaturesResources.Alias_ambiguous_type_0, diagnosticNode.ToString());
-                var groupingCodeAction = new CodeActionWithNestedActions(groupingTitle, codeActionsBuilder.ToImmutable(), isInlinable: true);
+                var groupingTitle = string.Format(CodeFixesResources.Alias_ambiguous_type_0, diagnosticNode.ToString());
+#pragma warning disable RS0005 // Do not use generic 'CodeAction.Create' to create 'CodeAction'
+                var groupingCodeAction = CodeAction.Create(groupingTitle, codeActionsBuilder.ToImmutable(), isInlinable: true);
+#pragma warning restore RS0005 // Do not use generic 'CodeAction.Create' to create 'CodeAction'
```

Comment:
seems problematic.

Reply:
@CyrusNajmabadi I don't think there is a problem regarding the suppression.

The analyzer will be removed soon anyway when @mavasani PR that fixes telemetry is merged.

There are already few suppressions also anyway.